even_fib_nums.js

```diff
@@ -5,10 +5,35 @@
  * @param  {Number} maxFibValue
  * @return {Number} sum
  */
-function _sumFibs( maxFibValue ) {
-  var sum = 0;
 
-  // do your work here
+const _sumFibs = ( maxFibValue ) => {
+  const fibs = [  ];
+  let sum = 0;
+
+  const fibsUpTo = ( num ) => {
+    let current = 1;
+    let previous = 0;
+    let temp;
+    
+    while ( current < num ) {
+      temp = current; // current = temp = 5, previous = 3;
+      current += previous; // current = 8, temp = 5, previous =3;
+      previous = temp; // current = 8, previous = 5, temp = 5;
+      console.log( `**temp is: ${ temp }, current is: ${ current }, previous is: ${ previous}` );
+      console.log( '\n' );
+      fibs.push( current );
+    }
+    return fibs;
+  }
+
+  fibsUpTo( maxFibValue );
+  console.log( fibs );
+
+  for ( let i = 0; i < fibs.length; i++ ) {
+    if ( fibs[i] % 2 === 0 ) {
```

Comment:
Try to think of how you could do this in the while loop above so you don't need to traverse the array of numbers twice.